e2e_playwright/st_dataframe_selections_test.py

```diff
@@ -375,6 +375,8 @@ def test_selection_state_remains_after_unmounting(
 ):
     """Test that the selection state remains after unmounting the component."""
     canvas = _get_multi_row_and_column_select_df(app)
+    expect_canvas_to_be_visible(canvas)
```

Comment:
This test failed within this PR (not the nightly as mentioned in the PR description)